### ISSUE TITLE
Enhancement. Don't use delay feature if that's required.

### ISSUE THREAD

**Issue by Sadzurami** [mcollina/close-with-grace#16]
In some cases will be cool to just wait until callback will be resolved, without timer.

Right now the only way to achieve this is set delay to max 32bit integer (setTimeout restriction).
```js
closeWithGrace({ delay: 2 ** 31 - 1 }, () => somethingUseful())
```

24.8 days (32bit integer) is enough for most cases, but will be better just drop delay feature if we won't it.
Passing `null` or `false` to delay property for example (not `undefined`, because it breaks default behavior).
```js
closeWithGrace({ delay: null }, () => somethingUseful())
```

I can make PR if it suits for lib.

**Comment by mcollina** [comment]
Thanks for reporting! Would you like to send a Pull Request to address this issue? Remember to add unit tests.

**Comment by Sadzurami** [comment]
released in v2.0.0
